js/notify/notifyImpl.js

```diff
@@ -241,4 +246,46 @@ function Queue(emitter) {
     return modifiedEmitter;
 }
 
+/**
+ * Replace HTML tags <> from messages test to prevent
+ * HTML injection
+ *
+ * @param string
+ * @return {void | string | *}
+ */
+function replaceHTMLTags(string) {
+
+    if (typeof string !== 'string') {
+        return null;
+    }
+
+    const tagsToReplace = {
+        '<': '&lt;',
+        '>': '&gt;'
+    };
+
+    function replaceTag(tag) {
+        return tagsToReplace[tag] || tag;
+    }
+
+    return string.replace(/[<>]/g, replaceTag);
+}
+
+/**
+ * Replace strong HTML tags from the string
+ *
+ * @param string
+ * @return {void | string | *}
+ */
+// TODO(KiranNiranjan): Can be removed after 2-3 sprints
+function replaceStrongTag(string) {
```

Comment:
Same here.

Reply:
@VikasShashidhar No, we need this to be called for backward compatibility reasons.

Reply:
I was actually talking about the ToDo comment above that.